adoc/retail_migr_chap_prepare.adoc

```diff
@@ -0,0 +1,26 @@
+[[retail_migr.chap.prepare]]
+= Prepare to Migrate
+
+include::entities.adoc[]
+
+
+[[retail.sect.prepare]]
+== Before you Begin
+
+Before you begin migration, there are a few things you need to consider:
+
+* Something something
+* (environment, planning, etc)
+* ...
+
+
+
+[[retail.sect.prepare.scenarios]]
+== Migration Scenarios
+
+This document contains several different migration scenarios.
+Ensure you pick the most appropriate migration scenario for your environment.
+
+If you are migrating from any version of {slepos} to {smr}, and you do not have a current {susemgr} installation, see: xref:retail_migr_slepostosuma.adoc#retail_migr.chap.slepostosuma[SLEPOS to new SUSE Manager]
```

Comment:
remove `current` as it may be 3.1 or even older.
`and you do not have a current {susemgr} installation` ->
`and you do not have a {susemgr} installation`